test/db/cmd/cmd_pd

```diff
@@ -675,7 +675,38 @@ e asm.lines.fcn=false
 pdf
 EOF
 EXPECT=<<EOF
-/ 13: fcn2.0x004017c0 ();
+/ fcn2.0x004017c0 ();
+|           0x004017c0      51             push  ecx
+|           0x004017c1      89e1           mov   ecx, esp
+|           0x004017c3      83c108         add   ecx, 8
+|           0x004017c6      3d00100000     cmp   eax, 0x1000
+\       ,=< 0x004017cb      7210           jb    0x4017dd
+
+fcn2.0x004017c0 ();
+          0x004017c0      51             push  ecx
+          0x004017c1      89e1           mov   ecx, esp
+          0x004017c3      83c108         add   ecx, 8
+          0x004017c6      3d00100000     cmp   eax, 0x1000
+      ,=< 0x004017cb      7210           jb    0x4017dd
+EOF
+RUN
+
+NAME=pdf fcnline with size
+FILE=bins/pe/a.exe
+CMDS=<<EOF
+e asm.bytes=true
+e asm.lines.fcn=true
+s 0x004017c0
+af+ 0x004017c0 fcn2.0x004017c0
+afb+ 0x4017c0 0x4017c0 13 0x4017dd 0x4017cd
+pdf
+?e
+e asm.lines.fcn=false
+e asm.fcn.size=true
```

Comment:
@XVilka I did here
I can add a few more of needed

Reply:
Ah, didn't notice. No worries then, should be fine.